saas/models/saas_operator.py

```diff
@@ -52,3 +54,33 @@ def get_db_name(self, db):
         # TODO: use mako for url templating
         self.ensure_one()
         return self.db_url_template.format(db_id=db.id)
+
+    def get_auth_urls(self, db):
+        self.ensure_one()
+        if self.type == 'local':
+            master_url = self.env['ir.config_parameter'].get_param('web.base.url')
+            build_url = self.get_db_url(db)
+        return master_url, build_url
+
+    def build_execute_kw(self, build, code):
+        if self.type == 'local':
+            db = sql_db.db_connect(build.name)
+            registry(build.name).check_signaling()
```

Comment:
Can you explain what is for?

Reply:
Most likely, such a construction will not be at all after the refactor, I found a more beautiful way